client/src/components/fetchRequests/fetchPlayer.js

```diff
@@ -0,0 +1,16 @@
+const fetchPlayer = async (playerName) => {
+  try {
+    const response = await fetch(`/api/v1/players3rdParty?playerName=${playerName}`)
+    if (!response.ok) {
+      const errorMessage = `${response.status} (${response.statusText})`
+      const error = new Error(errorMessage)
+      throw error
+    }
+    const playerData = await response.json()
+    return playerData
+  } catch (error) {
+    console.error(`Error in fetch: ${error.message}`)
+  }
+}
+
+export default fetchPlayer
```

Comment:
nice job separating htis into a separate file!!! 

I would change `players3rdParty` to something more standard, like `/api/v1/<name-of-your-api>-players...`

where<name-of-your-api> is the api your are making a got request to. otherwise this looks a bit odd, even to employers. our users dont need to know that this is necessarily third party info